Gruntfile.coffee

```diff
@@ -39,6 +39,7 @@ module.exports = (grunt) ->
 			"sprites"
 			"css"
 			"js"
+			"string-replace"
```

Comment:
Maybe better to stick with `string-replace` even if we can't add the target. Looking at the project it looks like it was more recently maintained and updated for Grunt 1.0 where the other wasn't